CK3toEU4/Source/Mappers/ReligionMapper/ReligionMapper.cpp

```diff
@@ -27,20 +27,23 @@ mappers::ReligionMapper::ReligionMapper(std::istream& theStream)
 void mappers::ReligionMapper::registerKeys()
 {
 	registerKeyword("link", [this](const std::string& unused, std::istream& theStream) {
-		const ReligionMapping theMapping(theStream);
-		for (const auto& ck3Religion: theMapping.getCK3Religions())
-		{
-			CK3toEU4ReligionMap.insert(std::make_pair(ck3Religion, theMapping.getEU4Religion()));
-		}
+		for (const ReligionMapping theMapping(theStream); const auto& ck3Religion: theMapping.getCK3Religions())
```

Comment:
I think this would be more readable:
```suggestion
		for (const auto& ck3Religion: ReligionMapping(theStream).getCK3Religions())
```

Reply:
nope, need theMapping to grab school a bit later.

Reply:
Ah, correct